src/TeamData.elm

```diff
@@ -37,22 +37,21 @@ view : Model -> Element.Element Msg
 view model =
     column
         [ Background.color sky
-        , Border.color black
         , padding 50
-        , spacing 20
-        , widthXY 5 5
-        , rounded 10
+        , spacing 40
         , centerX
         , centerY
+        , rounded 20
         ]
         [ textInput model.scouterName ScouterInput "Scouter's name"
-        , radioRow
+        , radio
             [ padding 10
             , spacing 20
+            , Font.size 60
             ]
             { onChange = Station
             , selected = Just model.station
-            , label = Input.labelAbove [] (text "Which station?")
+            , label = Input.labelAbove [ Font.size 60, padding 10, spacing 20 ] (text "Which station?")
```

Comment:
I haven't checked yet how does it look, is the _font 60_ instead of 20 because that an actual font 60 looks better (which I doubt), or did it look smaller then an actual font 20 and the font 60 looks like a font 20 suppose' to look?